test/unit/modules/modules.spec.ts

```diff
@@ -209,3 +209,56 @@ test("Export Equals", () => {
 
     expect(result).toBe(true);
 });
+
+const reassignmentTestCases = [
+    "x = true",
+    "[x] = [true]",
+    "[[x]] = [[true]]",
+    "({ x } = { x: true })",
+    "({ y: x } = { y: true })",
+    "({ x = false } = { x: undefined })",
+];
+
+test.each(reassignmentTestCases)("export specifier with reassignment afterwards (%p)", reassignment => {
+    util.testModule`
+            let x: any = false;
+            export { x };
+            ${reassignment};
+        `.expectToMatchJsResult();
+});
+
+test.each(reassignmentTestCases)("export specifier fork (%p)", reassignment => {
+    util.testModule`
+        let x: any = false;
+        export { x as a };
+        export { x as b };
+        ${reassignment};
+    `.expectToMatchJsResult();
+});
+
+test.each(["(() => { let a = 2; a = 3 })()", "{ let a = 2; a = 3 }"])(
```

Comment:
Do we really need to test both of these, given how similar syntactically they are?